lib/ld-key-pair.js

```diff
@@ -28,6 +30,7 @@ class LDKeyPair {
    */
   constructor(options) {
     this.injector = options.injector || new Injector();
+    this.algorithm = options.algorithm;
```

Comment:
I don't think we have a need for `algorithm`, do we?

Reply:
Agreed, we don't need it now.